src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
Good chat around this. Once again this was another copy pasta. I typically do the following (avoiding the ternary for readability):

```php
public static function instance() {
  if ( ! self::$instance ) {
    self::$instance = new self;
  }

  return self::$instance;
}
```